src/tools/schema2kotlin.ts

```diff
@@ -7,15 +7,11 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
-
-
-const description = `Generates Kotlin code from Schemas for use in wasm particles.`;
-
-import {Schema2Base, typeSummary} from './schema2base.js';
-
+import {Schema2Base} from './schema2base.js';
+import {Schema} from '../runtime/schema.js';
 
 // https://kotlinlang.org/docs/reference/keyword-reference.html
-// var keywords = [...document.getElementsByTagName('code')].map((x) => x.innerHTML);
+// [...document.getElementsByTagName('code')].map(x => x.innerHTML);
```

Comment:
Thanks!

Reply:
:)  I borrowed this for the C++ impl - good idea to link a reference.